lib/storage/s3.js

```diff
@@ -59,7 +98,22 @@ module.exports = function() {
       }
       https = options.https;
       cachingTime = options.cachingTime;
-      return callback(null);
+      // CORS
+      client.getBucketCors(getCorsConfig, function (err, data) {
+        if (err) {// an error occurred
+          console.log(err, err.stack);
+          return callback(null);
+        } else { // successful response
+          if (data.CORSRules !== putCorsConfig.CORSConfiguration.CORSRules) {
+            client.putBucketCors(putCorsConfig, function (err, data) {
+              if (err) {// an error occurred
```

Comment:
This should just be passing the callback to putBucketCors as the second argument, I think it's probably leftover debugging code. It doesn't make sense to ignore the error and continue.